### ISSUE TITLE
1.18.2 forge support?

### ISSUE THREAD

**Issue by alsodoze** [zbx1425/world-comment#3]
any chance of this mod to support forge 1.18.2? amazing mod by the way!

**Comment by zbx1425** [comment]
Yeah that's on the planned list, but I might come back to that after a while
Supporting multiple versions isn't easy, and as it seems I've took a quite stupid path to do that, just look at the mess I made to support both 1.20.1 and 1.19.2 xd
![图片](https://github.com/zbx1425/world-comment/assets/25097028/172c61f8-38e3-4247-85e6-513c8918c371)
![图片](https://github.com/zbx1425/world-comment/assets/25097028/bb042fd5-c50c-4e5b-b8e6-7b83766e2de3)

**Comment by alsodoze** [comment]
> Yeah that's on the planned list, but I might come back to that after a while Supporting multiple versions isn't easy, and as it seems I've took a quite stupid path to do that, just look at the mess I made to support both 1.20.1 and 1.19.2 xd ![图片](https://private-user-images.githubusercontent.com/25097028/292763075-172c61f8-38e3-4247-85e6-513c8918c371.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDM1NjIyNzksIm5iZiI6MTcwMzU2MTk3OSwicGF0aCI6Ii8yNTA5NzAyOC8yOTI3NjMwNzUtMTcyYzYxZjgtMzhlMy00MjQ3LTg1ZTYtNTEzYzg5MThjMzcxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjI2VDAzMzkzOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM5OWExNzE2ZjM2NTRiZDhiMDlkNTY4YWNiYmU2MjQ3NzgxMDUwZTQ0MmY5YmYzYThhZTEwYzkzZjZkYjNmYmMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.UNQk3Ikpp3viwTUlb7FzIpsnLGakGNhxr96YBHbisCc) ![图片](https://private-user-images.githubusercontent.com/25097028/292763118-bb042fd5-c50c-4e5b-b8e6-7b83766e2de3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDM1NjIyNzksIm5iZiI6MTcwMzU2MTk3OSwicGF0aCI6Ii8yNTA5NzAyOC8yOTI3NjMxMTgtYmIwNDJmZDUtYzUwYy00ZTViLWI4ZTYtN2I4Mzc2NmUyZGUzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjI2VDAzMzkzOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWIxMDk3NDA5NTAyZjllMTBmNDRjZTY5NDU5YjBmYTJiYWZlN2FlMTAwODI3MDM0ZTY3OGQwYTk0MjllMWQwYTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.aoHLpXGwkY-hOGahS6y9_59mH3qBVF-2OLYWXkJj9kU)

ok thanks, take your own pace!

**Comment by zbx1425** [comment]
Lemme open it till it's addressed
